gateways/clients/hathor_core_client.py

```diff
@@ -71,7 +75,7 @@ async def get(
                             status=response.status,
                             body=await response.text(),
                         )
-                    return await response.json()
+                    return await response.json(content_type=content_type)
```

Comment:
question(blocking): What is the purpose of this `content_type`?

Reply:
It acts as the expected Content-Type for aiohttp. It will check whether the response Content-Type matches the expected one, and raise an error if not.

Their default is `application/json`, so they reject parsing as JSON anything that has a different Content-Type.

My change will allow us to avoid aiohttp raising an error when hathor-core returns a `text/html` from the `/health` endpoint.

I will pass `None` to it, which basically disables this check. I'm doing this instead of passing `text/html` to make sure we won't break the explorer-service after https://github.com/HathorNetwork/hathor-core/pull/1110 gets deployed